### PR TITLE
Remove pragma once

### PR DESCRIPTION
**Pull request by tiagomacarios**
Remove pragmas and expanded guards to reduce chance of collision

### CODE REVIEW

File: include/gsl/multi_span [hunk 1]
```diff
@@ -14,10 +14,8 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#pragma once
-
-#ifndef GSL_MULTI_SPAN_H
-#define GSL_MULTI_SPAN_H
+#ifndef GUIDELINE_SUPPORT_LIBRARY_SPAN_H
```
Comment (neilmacintosh) [inline include/gsl/multi_span]: Shouldn't this be `GUIDELINE_SUPPORT_LIBRARY_MULTI_SPAN_H`?

Comment (tiagomacarios) [reply include/gsl/multi_span]: Fixed

**Comment by tiagomacarios** [comment]
@neilmacintosh Ill withdrawn this so it is not lingering around. If we get collisions with the guard name I can put it back together

**Comment by neilmacintosh** [comment]
Thanks @tiagomacarios sounds good.

**Pull request closed without merging**